### DISCUSSION TITLE
Even after providing the Authentication Key, The chaos dashboard is in the status of Unauthorized.

### DISCUSSION THREAD

**Discussion by Gittysri** [chaos-mesh/chaos-mesh#4309] [Q&A]
## Bug Report

Even after generating and providing the RBAC Token and using it in the chaos dashboard to login. The status of the chaos dashboard still shows Unauthorized

**What version of Chaos Mesh are you using?**
v2.1.8

**What did you do? / Minimal Reproducible Example**
We are using the chaos mesh with the version of 2.1.8 . We are facing many issues in the chaos dashboard. Like we can't rerun the same experiment again , even after using RBAC token to login, it is still in Unauthorized status .

**What did you expect to see?**
After providing the RBAC token for login with dashboard. The dashboard should be authorized .

**What did you see instead?**
But after providing the RBAC token for login with dashboard. The dashboard status in Unauthorized Status , due to this no experiments can be created.

**Output of chaosctl**
![MicrosoftTeams-image (3)](https://github.com/chaos-mesh/chaos-mesh/assets/69222533/d1938373-58e3-4bad-a403-7f8f5ae5b76a)

**Comment by STRRL** [comment]
First of all, it seems the RBAC might be not configured well, so it you could provide the related resources it might be helpful.

And you could use the same token for configuring kubectl and take a look why the RBAC role not work.
